src/Domain/Event/EventBufferInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Domain\Event;
+
+use Broadway\Domain\DomainMessage;
+
+interface EventBufferInterface
```

Comment:
Sound pure infrastructure and not sure why we need that.

Reply:
The more I think about it, the more I have to agree with you. I'll remove it.